packages/asc-ui/src/components/TopTaskButton/TopTaskButtonStyle.ts

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import styled from '@datapunt/asc-core'
+import { themeSpacing, themeColor } from '../../utils'
+
+import IconStyle from '../Icon/IconStyle'
+
+export interface Props {
+  icon: React.ElementType
+  title?: string
+  content?: string
+  onDarkBackground?: boolean
+  width: string
+  className: string
+}
+
+const defaultProps = {
+  onDarkBackground: false,
+  width: '220px',
+}
+
+export default styled.div<Props>`
+  width: ${({ width = defaultProps.width }) => width};
```

Comment:
I think this shouldn't be here. the user of the component should decide the width of the toptask (either by wrapping it in a grid, or extend it like `styled(TopTaskButton)'width: 300px;'`)

Reply:
you are right, I will remove that

Reply:
done